sandbox/apparmor/profile.go

```diff
@@ -118,8 +118,7 @@ var LoadProfiles = func(fnames []string, cacheDir string, flags AaParserFlags) e
 		return nil
 	}
 
-	// Use no-expr-simplify since expr-simplify is actually slower on armhf (LP: #1383858)
```

Comment:
Does this mean that removing `no-expr`simplify` will be faster for non-armhf architectures, and only result in slower performance on armhf?  I assume profile generation is slower in all cases, but is rule matching faster (except on armhf) after this change?

Reply:
Actually, armhf are the systems where we observed 10x memory and time explosion first.
`-O no-expr-simplify` meant that in most cases profile parsing was faster, but as Michael mentioned, this was back in 2014 when profiles were a lot simpler. 
Now with some complex profiles we generate, we observe 10x parser time and memory increase.
Profile referenced in the bug took 2mins and 40 seconds and 1+GB to parse on a device comparable to Pi4.